.travis.yml

```diff
@@ -5,4 +5,10 @@ before_script:
 - npm install -g grunt-cli
 - npm install -g bower
 - npm install --no-bin-links
-- bower install
\ No newline at end of file
+- bower install
+env:
+  global:
+  - secure: a8A7+rxGW9qfGPzfJHwoHZzvMiGxxmYfCyvpUMoNXqpTdIsT/vGV3nN2I/nHAKqh1iKMWyl1SNhuJqkljtE/1tmlX8927ZxQU09g4y17hulIO2NZvHkFfWXCfIj4uBZS2vKPuvAnGIaa5FIbY6D2U9tXuawSq5u8pKbOw8Bftz0=
+  - secure: Cv13GXmrbraNvQKl41eKIfgXlGLxZjP4mN7GxAR+nOWwnYmY0grZoI+JZwA31e5r6Tc8z28Bkb374epN9i3Y89G4uMVpWNXB/KVDshF0YlF4ReqgG6B/7+wYBJj7kRpISXTjArSNRF1DsbM1Ny+HWW7HGoArw02TAiVDmcPEMWY=
+addons:
+  sauce_connect: true
```

Comment:
I would look at trying to remove this. To my knowledge this is a feature offered by Travis in order to create a sauce tunnel. We already have this logic in our code so that we can execute tests on sauce VMs locally.